### ISSUE TITLE
tests are too verbose

### ISSUE THREAD

**Issue by pixelass** [stylelint/stylelint#1969]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

question

> Which rule, if any, is this issue related to?

all tests

> What CSS is needed to reproduce this issue?

this is a dev issue. no CSS needed

> What stylelint configuration is needed to reproduce this issue?

this is a dev issue. no CSS needed

> Which version of stylelint are you using?

`7.4.2`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

`npm test` || `./node_modules/.bin/babel-tape-runner src/rules/,SOME_RULE>/__tests__/index.js`

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

this is a dev issue. no CSS needed

> What did you expect to happen?

only failing test should be logged.

optional: error should be visually marked (red)

> What actually happened (e.g. what warnings or errors you are getting)?

there are over 23000 tests. all are logged. even when working in a single rule there are often over 90 tests. Finding the failing tests is almost impossible.

To find a failing test I would have to:

* copy the content in my text editor
* look at tape/api to find out how a failed test looks
* find failed test in pasted report

current output after adding two tests does not help me because I have to check all 97 reports for those 2 failures

```shell
1..97
# tests 97
# pass  95
# fail  2
```

**Comment by evilebottnawi** [comment]
@pixelass use pipe to file `npm test || ./node_modules/.bin/babel-tape-runner src/rules/,SOME_RULE>/__tests__/index.js > failed`

**Comment by pixelass** [comment]
@evilebottnawi thx a bunch.. will test it

**Comment by davidtheclark** [comment]
@pixelass This is how Tap test results work. 

You can find the failing tests via Ctrl+F "not ok". That's how I do it, and it works fine. 

Alternately, you can try out the many [pretty tape reporters](https://github.com/substack/tape#pretty-reporters) to get an output you like more. 

Finally, I'll add that if anybody wants to put in the work experimenting with alternate test runners for our rules, go for it! Last I tried, the current set-up was significantly faster than mocha; AVA just crashed my computer every time; and I couldn't get Jest to work, some syntax issue got in the way.

**Comment by pixelass** [comment]
@evilebottnawi Your suggestion did not work. It outputs the content to a file but I still have to search for them. The output did help though (totally forgot that I can simply print it to a file over cli)

@davidtheclark yeah, I figured. That's why I added the remark "I'm not familiar with tape".
I was really only asking to save time so looking for a reporter is the last thing I'll do. Thx for the info though.
